### PR TITLE
NO-ISSUE: Update to dev docs regarding feature-usage

### PR DESCRIPTION
**Pull request by sagidayan**
# Assisted Pull Request

## Description
Added a note under `How to add a usage entry`

> Since we also maintain a feature support list (ref: featuresupport/support_levels_list.go),
> If you do add a new constant to usage/consts.go, Please add the ID into swagger: `#/definitions/feature-support-level`.
> The feature ID is CAPITAL_SNAKE_CASE_OF_CONST. You can refer to `usage/manager.go:UsageNameToID` to see
> how the ID is generated.
> Update the feature support level if needed.

## List all the issues related to this PR

- [ ] New Feature
- [ ] Bug fix
- [ ] Tests
- [ ] Documentation
- [ ] CI/CD 

## What environments does this code impact?

- [ ] Automation (CI, tools, etc)
- [ ] Cloud
- [ ] Operator Managed Deployments
- [x] None

## How was this code tested?

- [ ] assisted-test-infra environment
- [ ] dev-scripts environment
- [ ] Reviewer's test appreciated
- [ ] Waiting for CI to do a full test run
- [ ] Manual (Elaborate on how it was tested)
- [x] No tests needed

## Assignees

/cc @
/cc @

## Checklist

- [ ] Title and description added to both, commit and PR.
- [ ] Relevant issues have been associated (see [CONTRIBUTING] guide)
- [ ] Reviewers have been listed
- [ ] This change does not require a documentation update (docstring, `docs`, README, etc)
- [ ] Does this change include unit-tests (note that code changes require unit-tests)

## Reviewers Checklist

- [ ] Are the title and description (in both PR and commit) meaningful and clear?
- [ ] Is there a bug required (and linked) for this change?
- [ ] Should this PR be backported?

[Kubernetes community documentation]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#commit-message-guidelines
[CONTRIBUTING]: https://github.com/openshift/assisted-service/blob/master/CONTRIBUTING.md

### CODE REVIEW

File: docs/dev/feature-usage.md [hunk 1]
```diff
@@ -19,6 +19,16 @@ the customer added.
 
 ***SetUsage*** is a helper function. When calling it you should specify the following parameters: 
 
+> Important note:
```
Comment (romfreiman) [inline docs/dev/feature-usage.md:22]: @slaviered lets add how to verify it in ELK?

Comment (slaviered) [reply docs/dev/feature-usage.md:22]: OK. When the links of ELK are final (they are on transition now).

**Pull request merged**